test/transforms.jl

```diff
@@ -685,6 +685,22 @@
     @test t == tₒ
   end
 
+  @testset "Coerce" begin
+    x1 = [1.0, 2.0, 3.0, 4.0, 5.0]
+    x2 = [1.0, 2.0, 3.0, 4.0, 5.0]
+    x3 = [5.0, 5.0, 5.0, 5.0, 5.0]
+    t = Table(;x1, x2, x3)
+
+    # apply test
+    n, c = apply(Coerce(:x1=>Count, :x2=>Count), t)
+    @test elscitype(n.x1) == Count && elscitype(n.x2) == Count
+
+    # revert test
+    n, c = apply(Coerce(:x1=>Count, :x2=>Count), t)
+    tₒ = revert(Coerce(:x1=>Count, :x2=>Count), n, c)
+    @test elscitype(tₒ.x1)==elscitype(t.x1) && elscitype(tₒ.x2)==elscitype(t.x2)
+  end
+
```

Comment:
Please make sure that this test set is also showing up before the Identity transform. Be consistent because that pays off in the long run.